frontend/public/kubevirt/components/vm.jsx

```diff
@@ -1,6 +1,7 @@
 import * as _ from 'lodash-es';
 import React, { Component, Fragment } from 'react';
 
+import { ResourceEventStream } from '../../components/events';
```

Comment:
Can you please indirect the reference via `frontend/public/kubevirt/components/okdcomponents.jsx`?

Reply:
sure. I also changed it in modals which were referencing `../../../module/k8s` to `../../module/okdk8s`